xcube_server/controllers/tiles.py

```diff
@@ -1,15 +1,22 @@
 import time
 from typing import Dict, Any
 
+import matplotlib
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt
```

Comment:
Wrong API,  `matplotlib.pyplot` is a stateful module. Use it in interactive envs only.